examples/benchmarks/bubblebench/sharedtree/package.json

```diff
@@ -36,7 +34,6 @@
 		"test": "npm run test:jest",
 		"test:jest": "jest",
 		"test:jest:verbose": "cross-env FLUID_TEST_VERBOSE=1 jest",
-		"tsc": "tsc",
```

Comment:
Confirming we want this to be ESM only?

Reply:
Oh I guess it already was. Nvm

Reply:
We find all kind of fun things in this effort. :)